infrastructure/books/books.go

```diff
@@ -24,7 +24,7 @@ func (r *Repository) GetByID(id string) (*model.Book, error) {
 	var book model.Book
 	result := r.db.Preload("Covers").
 		Preload("Authors").
-		Where("open_library_id = ?", "/works/"+id).
+		Where("open_library_id = ?", id).
```

Comment:
So we’re removing `/works/` from the ID? I like that.

Reply:
I have to do a bit more research into the different OL ids, I think they are opaque in nature, e.g. all works are OL\d+W, all authors are OL\d+A. What I did here is just make the interface the same, you pass in /works/OL\d+W, which is what openlibrary gives for the ID of works. We already had authors working with /authors/OL\d+A as its parameter. We're just being consistent.